modernizer-maven-plugin/pom.xml

```diff
@@ -0,0 +1,119 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Did you create this via `git mv`?  It doesn't seem to retain the history.

Reply:
This is a known github issue: https://github.com/isaacs/github/issues/900